apps/landingPage/src/components/info.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+
+import React from 'react';
+import { Heading, Text } from '@kiwicom/orbit-components/lib/';
+import styled from 'styled-components';
+
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
+import ios from '../images/ios.png';
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
+import android from '../images/android.png';
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
+import web from '../images/web.png';
+
+type Props = {|
+  +iframeWidth: number,
+|};
+
+export default ({ iframeWidth }: Props) => (
+  <>
+    <ContainerTop>
+      <Header>
+        <Heading>It's cross-platform</Heading>
+        <Text size="large">Works on IOS , Android and on the web!</Text>
+      </Header>
+      <ContainerShowcase width={iframeWidth}>
+        <Titles>
+          <Heading type="title2">IOS</Heading>
+          <Heading type="title2">Android</Heading>
+          <Heading type="title2">Web</Heading>
+        </Titles>
+        <div style={{ display: 'flex' }}>
+          <img src={ios} alt="ios" height={500} />
+          <img src={android} alt="android" height={500} />
+          <img src={web} alt="web" height={500} />
+        </div>
+      </ContainerShowcase>
+    </ContainerTop>
+    <ContainerInfo>
+      <ContainerHeaderBottom>
+        <Heading>It has these new features</Heading>
+      </ContainerHeaderBottom>
+      <ContainerInfoBottom width={iframeWidth}>
+        <Info>
+          <Text>
+            {' '}
```

Comment:
Is that space necessary?

Reply:
It was auto fix, I guess. Will remove it when changing to new design